renderer/components/onboard/Sections.js

```diff
@@ -139,18 +154,43 @@ const QuizQuestion = ({qNum, question, onTrue, onFalse}) => (
         </FalseButton>
       </Flex>
     </div>
-  </QuizModal>
+  </React.Fragment>
 )
 
+const Animation = ({ okay }) => {
+  const animationData = okay ? tickAnimation : crossAnimation
+  return (
+    <Lottie
+      width={300}
+      height={300}
+      options={{
+        loop: false,
+        autoplay: true,
+        animationData: animationData,
+        rendererSettings: {
+          preserveAspectRatio: 'xMidYMid slice'
+        },
+        eventListeners: [{
+          eventName: 'complete',
+          callback: () => this.onAnimatonComplete(),
```

Comment:
Could it be that this is not firing due to the fact that inside of this scope `this` is referring to the `{}` object instead of the `Animation` class instance?

Maybe you want to copy `onAnimationComplete` onto the stack before passing it as a callback to this?

Reply:
Ah. I don't think that is the problem. I missed moving out the callback along with the `<Lottie ... />`. But it didn't work even when it was inside. I have reverted to that version now.

Reply:
Turns out `eventListeners` is a prop to `<Lottie>` :man_facepalming: